04-spatial-operations.qmd

```diff
@@ -473,7 +487,8 @@ as shown in @fig-nz-and-grid.
 #| fig-cap: The nz layer and a regular grid of rectangles
 
 base = grid.plot(color='none', edgecolor='grey')
-nz.plot(ax=base, column='Population', edgecolor='black', legend=True);
+nz.plot(ax=base, column='Population', edgecolor='black', legend=True, cmap='viridis_r');
```

Comment:
I'd recommend to use the reversed color ramps, so that high values are represented by darker colors

Reply:
Agree having just tested in codespaces now.